### ISSUE TITLE
3.2. Amplifier module

### ISSUE THREAD

**Issue by zhengchaobing** [MohitLamba94/Restoring-Extremely-Dark-Images-In-Real-Time#21]
I can not find the  Amplifier module when test the code

def run_test(model, dataloader_test, save_images):    
    with torch.no_grad():
        model.eval()
        for image_num, low in enumerate(dataloader_test):
            low = low.to(next(model.parameters()).device)            
            for amp in [1.0,5.0,8.0]:
                pred = model(amp*low)
                pred = (np.clip(pred[0].detach().cpu().numpy().transpose(1,2,0),0,1)*255).astype(np.uint8)
                imageio.imwrite(os.path.join(save_images,'img_num_{}_m_{}.jpg'.format(image_num,amp)), pred)
    return
